handsontable/src/dataMap/metaManager/metaSchema.js

```diff
@@ -4435,19 +4435,22 @@ export default () => {
      */
     tableClassName: undefined,
 
-    // TODO: add themeName description
     /**
-     * The `themeName` option lets you add CSS class names
+     * The `themeName` option allows enabling a theme by that name.
+     *
+     * If no `themeName` is provided, the table will use the classic theme (if the correct CSS files are imported).
+     *
+     * Read more:
+     * - [Themes](@/guides/styling/themes/themes.md)
      *
      * @memberof Options#
-     * @type {string|string[]}
+     * @type {string}
```

Comment:
I guess there is more?
```suggestion
     * @type {string | boolean | undefined}
```
Can I switch to the classic theme by passing `false`?